setup.py

```diff
@@ -58,8 +58,10 @@ def run(self):
 
         if self.type in ['unit', 'all']:
             cmd.append('python/Ganga/new_tests/Unit')
-            cmd.append('python/Ganga/Core')
-            cmd.append('python/Ganga/Runtime')
+            ## Ganga/Core finds zero tests
+            #cmd.append('python/Ganga/Core')
```

Comment:
I don't think it matters if there are zero tests in there at the moment. There likely will be tests at some point so it shouldn't hurt?